.github/dependabot.yml

```
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
```

LDWizard is being developed with yarn instead of NPM. Using NPM here would result in a conflict of lock files
```suggestion
  - package-ecosystem: "yarn"
```

---

GH [documents `npm` for both NPM and Yarn](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#package-ecosystem).

---

Ah, I assumed incorrectly that they would be incompatible, thanks for the information